static/js/components/CourseList.js

```diff
@@ -1,20 +1,34 @@
 import React from 'react';
 import _ from 'lodash';
 import Button from 'react-bootstrap/lib/Button';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
 
+import { toggleDashboardExpander } from '../actions';
 import {
   makeCourseStatusDisplay,
   makeCourseProgressDisplay,
+  makeRunStatusDisplay,
 } from '../util/util';
-import { STATUS_PASSED } from '../constants';
+import {
+  STATUS_PASSED,
+  DASHBOARD_COURSE_HEIGHT,
+  DASHBOARD_RUN_HEIGHT,
+} from '../constants';
 
 class CourseList extends React.Component {
   static propTypes = {
     dashboard: React.PropTypes.object.isRequired,
+    expander: React.PropTypes.object.isRequired,
+    dispatch: React.PropTypes.func.isRequired,
   };
 
+  toggleExpander(courseId) {
+    const { dispatch, expander } = this.props;
+    dispatch(toggleDashboardExpander(courseId, !expander[courseId]));
+  }
```

Comment:
if you declare `toggleExpander` as an arrow function we won't need to bind it to `this` later:

```js
toggleExpander = courseId => {
  ...
}
```